test/emailpassword/signoutFeature.test.js

```diff
@@ -13,10 +13,332 @@
+        );
+        assert(response.status === 404);
+    });
+
+    // Call the API without a session and it should return "OK"
+    it("test that calling the API without a session should return OK", async function () {
+        await startST();
+
+        STExpress.init({
+            supertokens: {
+                connectionURI: "http://localhost:8080",
+            },
+            appInfo: {
+                apiDomain: "api.supertokens.io",
+                appName: "SuperTokens",
+                websiteDomain: "supertokens.io",
+            },
+            recipeList: [EmailPassword.init(), Session.init()],
+        });
+
+        const app = express();
+
+        app.use(STExpress.middleware());
+
+        app.use(STExpress.errorHandler());
+
+        let response = await new Promise((resolve) =>
+            request(app)
+                .post("/auth/signout")
+                .end((err, res) => {
+                    if (err) {
+                        resolve(undefined);
+                    } else {
+                        resolve(res);
+                    }
+                })
+        );
+        assert(JSON.parse(response.text).status === "OK");
+        assert(response.header["set-cookie"] === undefined);
+    });
+
+    // Call the API without an expired access token session, and it should return TRY_REFRESH_TOKEN
+    it("test that calling the API wiithout the access token session, and it should return TRY_REFRESH_TOKEN", async function () {
+        await startST();
+
+        STExpress.init({
+            supertokens: {
+                connectionURI: "http://localhost:8080",
+            },
+            appInfo: {
+                apiDomain: "api.supertokens.io",
+                appName: "SuperTokens",
+                websiteDomain: "supertokens.io",
+            },
+            recipeList: [EmailPassword.init(), Session.init()],
+        });
+
+        const app = express();
+
+        app.use(STExpress.middleware());
+
+        app.use(STExpress.errorHandler());
+
+        let response = await signUPRequest(app, "random@gmail.com", "validpass123");
+        assert(JSON.parse(response.text).status === "OK");
+
+        let res = extractInfoFromResponse(response);
+
+        let response2 = await new Promise((resolve) =>
+            request(app)
+                .post("/auth/signout")
+                .set("Cookie", ["sAccessToken=" + "" + ";sIdRefreshToken=" + res.idRefreshTokenFromCookie])
+                .set("anti-csrf", res.antiCsrf)
+                .end((err, res) => {
+                    if (err) {
+                        resolve(undefined);
+                    } else {
+                        resolve(res);
+                    }
+                })
+        );
+        assert(response2.status === 401);
+        assert(JSON.parse(response2.text).message === "try refresh token");
+    });
+
+    //Call the API with an expired access and refresh token, and it should return "OK" (with clearing the cookies).
```

Comment:
Change the description of this test. You have not implemented what the description says.. Change the description to (NOT THE TEST, ONLY THE DESCRIPTION):
"Call the API with an expired access token, refresh, and call the API again to get OK and clear cookies"

Also, `await setKeyValueInConfig("refresh_token_validity", 2);` is useless since the units here is mins.. please remove it